types/rdfjs__parser-jsonld/index.d.ts

```diff
@@ -2,10 +2,15 @@ import { BaseQuad, DataFactory, Quad, Sink, Stream } from "@rdfjs/types";
 import { EventEmitter } from "events";
 import { Context } from "jsonld/jsonld-spec.js";
 
+export interface DocumentLoader {
+    load(url: string): Promise<unknown>;
+}
+
 export interface ParserOptions {
     baseIRI?: string | undefined;
     context?: Context | undefined;
     factory?: DataFactory | undefined;
+    documentLoader?: DocumentLoader | null;
```

Comment:
Looks like it should match https://github.com/rubensworks/jsonld-streaming-parser.js/blob/c17e43516a2d32e7221fb8fe0cede4b2200b576b/lib/JsonLdParser.ts#L591, so should be:

```suggestion
    documentLoader?: DocumentLoader | undefined;
```